src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js

```diff
@@ -462,7 +462,7 @@ Actions = {
 			showNegativeButton: false
 		};
 
-		Actions.spawnDialog("singleInput", dialogParams, onUserInput);
+		Actions.spawnDialog("inputAndSelection", dialogParams, onUserInput);
```

Comment:
Why did you change this into "inputAndSelection" since we're removing it?

Reply:
This was a mistake! Good catch!

Reply:
I think there is another one here: https://github.com/ChartIQ/finsemble-seed/pull/618/files#diff-db4c8003ad6f961267ff893ab63305e7R492

Reply:
Fixed.